pkgs/tools/audio/openai-whisper-cpp/default.nix

```
@@ -28,8 +29,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  buildInputs = [ SDL2 ] ++ lib.optionals stdenv.isDarwin [ Accelerate CoreGraphics CoreVideo ];
+  buildInputs = [ SDL2 ] ++ lib.optionals stdenv.isDarwin [ Accelerate CoreGraphics CoreML CoreVideo ];
 
+  WHISPER_COREML = lib.optionalString stdenv.isDarwin "1";
+  WHISPER_COREML_ALLOW_FALLBACK = lib.optionalString stdenv.isDarwin "1";
```

Does the following has any issue?

```suggestion
env = {
  WHISPER_COREML = lib.optionalString stdenv.isDarwin "1";
  WHISPER_COREML_ALLOW_FALLBACK = lib.optionalString stdenv.isDarwin "1";
  };
```

---

```suggestion
  env = lib.optionalAttrs stdenv.isDarwin {
    WHISPER_COREML = "1";
    WHISPER_COREML_ALLOW_FALLBACK = "1";
  };
```